health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserAccessMgmtTableWrapper.js

```
@@ -0,0 +1,232 @@
+
+const UserAccessMgmtTableWrapper = ({ role,}) => {
+  const { t } = useTranslation();
+  const tenantId = Digit.ULBService.getCurrentTenantId();
+  const [currentPage, setCurrentPage] = useState(1);
+  const [rowsPerPage] = useState(5);
+  const [totalRows, setTotalRows] = useState(0);
+  const [showPopUp, setShowPopUp] = useState(false);
+  const { campaignId, microplanId, key, ...queryParams } = Digit.Hooks.useQueryParams();
+
+
+  const { isLoading: isLoading, data: planAssignmentData, refetch: refetchPlanSearch} = Digit.Hooks.microplanv1.usePlanSearchEmployeeWithTagging({
```

Biome

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)